### PR TITLE
CSV model, row name suggestions

### PR DESCRIPTION
**Pull request by weronica**

### CODE REVIEW

**Comment by armant** [comment]
r @dzhang55

File: app/bulk_resource/views.py [hunk 1]
```diff
@@ -1,8 +1,11 @@
 import csv
```
Comment (armant) [inline app/bulk_resource/views.py]: nit: newline after this

File: app/bulk_resource/views.py [hunk 2]
```diff
@@ -1,8 +1,11 @@
...
+from datetime import datetime
 from flask import redirect, render_template, url_for
```
Comment (armant) [inline app/bulk_resource/views.py]: nit: newline after this

File: app/bulk_resource/views.py [hunk 3]
```diff
@@ -1,8 +1,11 @@
...
 from flask.ext.login import login_required
 from . import bulk_resource
```
Comment (armant) [inline app/bulk_resource/views.py]: nit: move ".", "..", and others start start with a symbol to before the first one that starts with a letter

File: app/bulk_resource/views.py (hunk 2 above)
Comment (weronica) [inline app/bulk_resource/views.py]: Done

File: app/bulk_resource/views.py (hunk 3 above)
Comment (weronica) [inline app/bulk_resource/views.py]: Done

File: app/admin/views.py [hunk 4]
```diff
@@ -1,18 +1,17 @@
...
+from . import admin
+from .. import db
+from ..decorators import admin_required
+from ..email import send_email
+from ..models import Role, User
 from forms import (
     ChangeUserEmailForm,
     NewUserForm,
     ChangeAccountTypeForm,
     InviteUserForm,
```
Comment (dzhang55) [inline app/admin/views.py]: Can this be alphabetical order? (also I think I actually fix this in my PR so hopefully that's okay)

File: app/bulk_resource/views.py [hunk 5]
```diff
@@ -25,4 +48,6 @@ def upload():
 @bulk_resource.route('/review')
 @login_required
 def review():
-    return render_template('bulk_resource/review.html')
+    csv_container = CsvContainer.query.first()
```
Comment (dzhang55) [inline app/bulk_resource/views.py]: This would also have to be associated with the current_user right?

File: app/admin/views.py (hunk 4 above)
Comment (weronica) [inline app/admin/views.py]: Done

File: app/bulk_resource/views.py (hunk 5 above)
Comment (weronica) [inline app/bulk_resource/views.py]: Yes, definitely. For now, I added a `TODO`. Something is broken with `current_user` at the moment...

**Pull request merged**